app/src/main/java/com/wut/indoornavigation/logic/graph/UnionFind.java

```diff
@@ -0,0 +1,8 @@
+package com.wut.indoornavigation.logic.graph;
+
+public interface UnionFind {
+    void union(Integer p);
+    void union(Integer p, Integer q);
+    boolean connected(Integer p);
+    boolean connected(Integer p, Integer q);
```

Comment:
staraj sie uzywac typow prostych, a i jak uzywasz pojedynczych liter to warto dodac komentarze nad funkcjami co to ma byc